examples/plugins/executor/hello/hello-executor-plugin-configmap.yaml

```
@@ -0,0 +1,62 @@
+# This is an auto-generated file. DO NOT EDIT
+apiVersion: v1
+data:
+  sidecar.address: http://localhost:4355
+  sidecar.container: |
+    args:
+    - |
+      import json
+      from http.server import BaseHTTPRequestHandler, HTTPServer
+
+
+      class Plugin(BaseHTTPRequestHandler):
+
+          def args(self):
+              return json.loads(self.rfile.read(int(self.headers.get('Content-Length'))))
+
+          def reply(self, reply):
+              self.send_response(200)
+              self.end_headers()
+              self.wfile.write(json.dumps(reply).encode("UTF-8"))
+
+          def unsupported(self):
+              self.send_response(404)
+              self.end_headers()
+
+          def do_POST(self):
+              if self.path == '/api/v1/template.execute':
+                  args = self.args()
+                  if 'hello' in args['template'].get('plugin', {}):
+                      self.reply({'node': {'phase': 'Succeeded', 'message': 'Hello template!'}})
+                  else:
+                      self.reply({})
+              else:
+                  self.unsupported()
+
+
+      if __name__ == '__main__':
+          httpd = HTTPServer(('', 4355), Plugin)
+          httpd.serve_forever()
+    command:
+    - python
+    - -c
+    image: python:alpine3.6
+    name: hello-executor-plugin
+    resources:
+      limits:
+        cpu: 200m
+        memory: 64Mi
+      requests:
+        cpu: 100m
+        memory: 32Mi
+    securityContext:
+      runAsNonRoot: true
+      runAsUser: 1000
+kind: ConfigMap
+metadata:
+  annotations:
+    workflow.argoproj.io/description: This is the "hello world" plugin
```

nit: 
```suggestion
    workflow.argoproj.io/description: This is the "hello-executor-plugin" plugin
```

feels more consistent since it is not named: `hello world`

---

intent is different, this is a "hello world", so I think name does not matter